freqtrade/data/dataprovider.py

```diff
@@ -520,7 +520,7 @@ def trades(
             return self._exchange.trades(
                 (pair, timeframe or self._config["timeframe"], _candle_type), copy=copy
             )
-        elif self.runmode in (RunMode.BACKTEST, RunMode.HYPEROPT):
+        elif self.runmode in (RunMode.BACKTEST, RunMode.HYPEROPT, RunMode.PLOT):
```

Comment:
i suspect using this as "else" (without additional condition) might be less error-prone.

i think (didn't test it rn) it'll currently still not work in webserver mode in this way (good for backtesting interactively ...) - which is really the recommended way to use plotting nowadays.

Reply:
in principle - i do agree with you - but in this case - it doesn't make much sense - as there's no harm in not considering this